### DISCUSSION TITLE
DKIM & CNAME are always showing Unverified

### DISCUSSION THREAD

**Discussion by XenyFord** [postalserver/postal#2013] [Help with using Postal] [answered]
Hello Team,
I have updated all the SPF and DKIM CNAME record as it shown in the postal admin, but still showing that DKIM & CNAME s are unverified. Please help me how to solve this. thank you in advance!

**Comment by willpower232** [comment, marked as answer]
You can confirm your records have definitely changed with whatsmydns.net.

You can use the `dig` command on the server to look up your records and confirm it can see the updated records.

If those are definitely correct, you may need to change this config
https://github.com/postalserver/postal/blob/9f0697f194209f5fae5e451ba8fb888413fe37fa/config/postal.defaults.yml#L17

**Reply by XenyFord** [reply]
Checked everything, and also added the said line, but no results as expected - still the same DKIM & CNAME are showing as unverified. But, it was working very perfect a few days ago.

**Reply by XenyFord** [reply]
So, do you want me to trash the complete sever and re-install the postal application again? OR please let me know the best solution.. If possible I will post the login details to you to have a look-into it..Pls let me know know on how to have a quick solution for this

**Reply by XenyFord** [reply]
Yes Mr. Will,
It's been working very perfectly as it was before, after I changed the said line to 'true' as given below;
`use_local_ns_for_domains: true`
Thanks a lot sir :)

**Reply by XenyFord** [reply]
Hi All,

Below are the exact commands to run/execute on your postal server(any linux machine) to look up your DNS records and confirm whether they are updated correctly;

# `dig +nocmd psrp.YOURDOMAINME +noall +answer`
# `dig +nocmd postal-yDyuaJ._domainkey.YOURDOMAIN.com txt +noall +answer`

Thanks to everyone who helped me! Especially, Heartfelt thanks to Mr. Will @willpower232
